web_src/js/components/DashboardRepoList.vue

```diff
@@ -471,7 +472,7 @@ export default sfc; // activate the IDE's Vue plugin
           <li class="tw-flex tw-items-center tw-py-2" v-for="org in organizations" :key="org.name">
             <a class="repo-list-link muted" :href="subUrl + '/' + encodeURIComponent(org.name)">
               <svg-icon name="octicon-organization" :size="16" class-name="repo-list-icon"/>
-              <div class="text truncate">{{ org.name }}</div>
+              <div class="text truncate">{{ defaultShowFullName && org.full_name ? org.full_name : org.name }}</div>
```

Comment:
It should be ``org.full_name ? `${org.full_name} (${org.name})` : org.name``

No need to introduce `defaultShowFullName` which doesn't seem to make sense.

Reply:
a60fce5aa8c6f96f44519a424409480c2babaf60